test/test_other.py

```diff
@@ -12768,13 +12768,11 @@ def test_cpuprofiler_closure(self):
 
   # Make sure that --memoryprofiler compiles with --closure 1
   def test_memoryprofiler_closure(self):
-    # TODO: Enable '-Werror=closure' in the following, but that has currently regressed.
-    self.run_process([EMCC, test_file('hello_world.c'), '-O2', '--closure=1', '--memoryprofiler'])
+    self.run_process([EMCC, test_file('hello_world.c'), '-O2', '--closure=1', '--memoryprofiler'] + self.get_emcc_args())
```

Comment:
was this meant to actually enable -Werror=closure?

Reply:
Ah yes, that is actually in `self.get_emcc_args()` so this change does add that, effectively.